app/ide-desktop/lib/content/esbuild-config.ts

```diff
@@ -122,6 +130,29 @@ export function bundlerOptions(args: Arguments): esbuild.BuildOptions {
             // We do not control naming of third-party options.
             esbuildPluginAlias({ ensogl_app: ensoglAppPath }),
             esbuildPluginTime(),
+            // This must run before the copy plugin so that the generated `tailwind.css` is used.
+            {
+                name: 'enso-generate-tailwind',
+                setup: build => {
+                    build.onStart(() => {
+                        const dest = path.join(outputPath, 'tailwind.css')
+                        const config = path.resolve(THIS_PATH, 'tailwind.config.ts')
+                        console.log(
+                            `Generating tailwind css from '${TAILWIND_CSS_PATH}' to '${dest}'.`
+                        )
+                        childProcess.spawnSync(`node`, [
+                            TAILWIND_BINARY_PATH,
+                            '-i',
+                            TAILWIND_CSS_PATH,
+                            '-o',
+                            dest,
+                            '-c',
+                            config,
+                            '--minify',
+                        ])
```

Comment:
I think my comment from my previous PR review was lost again here. I added here a comment that this is a really long expression and according to our style guide, we should split it into one-line exprs instead. I'd be thankful if we will not use anymore force pushes. TBH, I do not understand why they are used - for the last 10? years, I never used a force push in my PRs, so I kind of don't understand what forces us to use them here :)

Reply:
nope, it's [still here](https://github.com/enso-org/enso/pull/5976#discussion_r1142852535). there have been no force pushes since that comment.

i've been force pushing when:
- the commit history has diverged too much (between the old base branch, and the new base branch), so redoing the changes is much easier, or
- it's a minor formatting change, and it's early enough (say, 5 minutes) that there are no reviews, or
- i did it through a github web UI with a rebase. this is probably a mistake, i'll be using regular merges through the web UI instead when i do thi

Reply:
but yeah i'm still not sure how exactly to split this into one-line exprs - do i split the array into two parts? not sure that will be more readable...

Reply:
Sure! What about this?

```
const tailwind_opts = [TAILWIND_BINARY_PATH, '-i', TAILWIND_CSS_PATH]
const output_opts = ['-o', dest]
const config_opts = ['-c', config, '--minify']
childProcess.spawnSync(`node`, [tailwind_opts, output_opts, config_opts].flat())
```

Reply:
still too long, unfortunately... (the last line is 102 characters.)

i guess i'll just try extracting it to a top level variable

Reply:
Don't spend too much time on it. If it would not fit, lets just skip it :)